app/controllers/employees_controller.rb

```diff
@@ -41,6 +42,14 @@ def destroy
     end
   end
 
+
+  def employees_list
+    @employees = Employee.order(:email)
+    respond_to do |format|
+      format.json { render json: Employee.where('role != ? AND name like ?', 'admin',"%#{params[:q]}%" ) }
```

Comment:
use 'admin' constant
I think it is already defined in Employee.rb

Reply:
yes, have changed it. sumbal did it too.